src/main/java/seedu/duke/commands/project/ProjectDeleteCommand.java

```diff
@@ -23,20 +23,20 @@ public ProjectDeleteCommand(HashMap<String, String> params)
     }
 
     public void parse() throws DukeExceptions {
```

Comment:
Why is the try catch shifted from parse to executeCommand?

Reply:
Cos it only catches DukeException right and not NumberFormatException?

Reply:
actually in the parse function there will be nothing to catch.
If you realise the only thing the parse function does is extract the user input.
It does not process it, so no errors thrown